diff --git a/CMakeLists.txt b/CMakeLists.txt
index e0e2433..cd3e0fa 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -132,7 +132,7 @@ ELSE (WIN32)
 			INCLUDE_DIRECTORIES(${X11_INCLUDE_DIR})
 			INCLUDE_DIRECTORIES(/usr/include/freetype2)
 			ADD_DEFINITIONS(-DLINUX)
-			SET(JUCE_PLATFORM_SPECIFIC_LIBRARIES ${X11_LIBRARIES} asound freetype Xinerama)
+			SET(JUCE_PLATFORM_SPECIFIC_LIBRARIES ${X11_LIBRARIES} asound freetype)
 		ENDIF(APPLE)
 	ENDIF(UNIX)
 ENDIF(WIN32)
diff --git a/JuceLibraryCode/AppConfig.h b/JuceLibraryCode/AppConfig.h
index 16bebe0..78a392a 100644
--- a/JuceLibraryCode/AppConfig.h
+++ b/JuceLibraryCode/AppConfig.h
@@ -17,7 +17,7 @@
 //==============================================================================
 // [BEGIN_USER_CODE_SECTION]
 
-// (You can add your own code in this section, and the Introjucer will not overwrite it)
+#define JUCE_USE_XINERAMA	0
 
 // [END_USER_CODE_SECTION]
 
