--- a/JuceLibraryCode/modules/juce_core/maths/juce_MathsFunctions.h
+++ b/JuceLibraryCode/modules/juce_core/maths/juce_MathsFunctions.h
@@ -381,7 +381,8 @@ inline bool juce_isfinite (float value) noexcept
    #if JUCE_WINDOWS && !JUCE_MINGW
     return _finite (value) != 0;
    #else
-    return std::isfinite (value);
+    using namespace std;
+    return isfinite (value);
    #endif
 }
 
@@ -391,7 +392,8 @@ inline bool juce_isfinite (double value) noexcept
    #if JUCE_WINDOWS && !JUCE_MINGW
     return _finite (value) != 0;
    #else
-    return std::isfinite (value);
+    using namespace std;
+    return isfinite (value);
    #endif
 }
 
