From 9d4a7ddf1bb8092358c64af9c92ebdb846a4f9dc Mon Sep 17 00:00:00 2001
From: "FeRD (Frank Dana)" <ferdnyc@gmail.com>
Date: Tue, 9 Apr 2019 12:04:14 -0400
Subject: [PATCH] Remove X11 dependencies

---
 CMakeLists.txt | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 99e9769..0763341 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -150,13 +150,11 @@ ELSE (WIN32)
 			SET_PROPERTY(GLOBAL PROPERTY JUCE_LINUX "JUCE_LINUX")
 			SET(EXTENSION "cpp")
 
-			find_package(X11 REQUIRED)
-			find_package(Freetype REQUIRED)
 			find_package(ZLIB REQUIRED)
 			find_package(ALSA REQUIRED)
-			include_directories(${X11_INCLUDE_DIR} ${FREETYPE_INCLUDE_DIR} ${ZLIB_INCLUDE_DIR} ${ALSA_INCLUDE_DIR})
+			include_directories(${ZLIB_INCLUDE_DIR} ${ALSA_INCLUDE_DIR})
 			ADD_DEFINITIONS(-DLINUX)
-			SET(JUCE_PLATFORM_SPECIFIC_LIBRARIES ${X11_LIBRARIES} ${FREETYPE_LIBRARIES} ${ZLIB_LIBRARIES} ${ALSA_LIBRARIES})
+			SET(JUCE_PLATFORM_SPECIFIC_LIBRARIES ${ZLIB_LIBRARIES} ${ALSA_LIBRARIES})
 			SET(CMAKE_CXX_FLAGS " ${CMAKE_CXX_FLAGS} -std=c++11")
 		ENDIF(APPLE)
 	ENDIF(UNIX)
-- 
2.20.1

